stub/src/main/java/io/grpc/stub/Calls.java

```diff
@@ -188,7 +190,7 @@
   public static <ReqT, RespT> RespT blockingClientStreamingCall(
       Call<ReqT, RespT> call,
       Iterator<ReqT> clientStream) {
-    SettableFuture<RespT> responseFuture = SettableFuture.create();
+    GrpcFuture<RespT> responseFuture = new GrpcFuture<RespT>(call);
```

Comment:
It appears that UnaryStreamToFuture can just depend on AbstractFuture and then not care about which type of future is passed. Then, this code can use SettableFuture as before. There is some value to that in my mind, because it reduces (just a bit) how much the various stub implementations depend on each other.